armory/data/datasets.py

```diff
@@ -562,6 +562,22 @@ def wrapped(x):
     return wrapped
 
 
+def label_preprocessing_chain(*args):
```

Comment:
Yeah, I don't know a way to capture the monadic and dyadic forms in a way that isn't likely worse.  

Be nice to the next guy and perhaps call out in comment the difference between them? Could be "This function chains across (x, y) instead of just (x) as in preprocessing_chain.

Reply:
done